src/views/Home/components/NewsCard/index.js

```diff
@@ -5,12 +5,8 @@
 // May be a potential future feature, but not sure
 
 import React, { Component } from 'react';
-import Grid from '@material-ui/core/Grid';
-import CardHeader from '@material-ui/core/CardHeader';
-import Card from '@material-ui/core/Card';
-import Typography from '@material-ui/core/Typography';
 import { CardContent } from '@material-ui/core';
```

Comment:
```suggestion
```

Reply:
I will refactor these in a new pr to develop. Since this is purely stylistic though, I don't think it should hold up our push to master.